ui/src/containers/VolumePage.js

```diff
@@ -101,6 +104,16 @@ const VolumePage = (props) => {
     getVolumeListData(state, props),
   );
 
+  // If data has been retrieved and no volume is selected yet we select the first one
```

Comment:
Not sure we want to auto-select the first volume when we arrive `/volumes`. I am wondering is this what you want to achieve?